### DISCUSSION TITLE
Auto-update

### DISCUSSION THREAD

**Discussion by hello-smile6** [JaydenDev/freecat#72] [Electron]
@JaydenDev Creating a `stable` branch. Stable releases will be pushed to that branch, and the app will auto-update using it. Going to use https://github.com/electron/update-electron-app

**Comment by hello-smile6** [comment]
Never mind about the new branch.

**Comment by imdevarsh** [comment]
Auto updating will only work if the app is signed right?

**Reply by hello-smile6** [reply]
Yes. Could you make a GitHub Actions script to generate self-signed binaries, by chance?

**Reply by imdevarsh** [reply]
I don't have a signing certificate, and I am not going to contribute much anymore. Anyways, I don't have the money to spend $100 for a certificate. For now, I'd say that you can just make your own code to detect if an update is available and just give a notice to the user to get it from github themselves.

**Reply by hello-smile6** [reply]
Just do self-signed.

**Reply by imdevarsh** [reply]
Self signed? Try that on Windows, or Mac.

**Reply by hello-smile6** [reply]
That's all it needs for Windows, at least.
